vulnerabilities/tests/test_nvd.py

```diff
@@ -139,9 +140,18 @@ def test_to_advisories(self):
                     [
                         Reference(
                             url="http://code.google.com/p/gperftools/source/browse/tags/perftools-0.4/ChangeLog",  # nopep8
+                            scores=[],
```

Comment:
Is there any need to initialize this at all?
Same for below https://github.com/nexB/vulnerablecode/pull/290/files#diff-5bd858dbcd0bcd0a70d34d3b2c47bcc08ab731569df61e265711ad8c8bedcb6cR147

Reply:
Nope

Reply:
Fixed